lib/cypress/scoop_and_filter.rb

```diff
@@ -0,0 +1,60 @@
+module Cypress
+  class ScoopAndFilter
+    def initialize(measures)
+      @relevant_codes = codes_in_measures(measures)
+      @demographic_oids = ['2.16.840.1.113883.10.20.28.3.55', '2.16.840.1.113883.10.20.28.3.59', '2.16.840.1.113883.10.20.28.3.56']
+      @hqmf_oids_for_measures = []
```

Comment:
This line seems like it might be unnecessary since it is set to something different on the next line.